src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int MAX_CAPACITY = 10;
+    private Object[] array = new Object[MAX_CAPACITY];
+    private int size;
+
+    public void grow() {
+        Object[] newArray = new Object[size + size / 2];
+        for (int i = 0; i < size; i++) {
+            newArray[i] = array[i];
+        }
```

Comment:
https://github.com/mate-academy/jv-array-list/blob/master/checklist.md#use-systemarraycopy-to-move-your-array-elements

In all places where you copy elements use relevant predefined method